src/api/Animated/AnimatedImplementation.js

```diff
@@ -40,9 +42,11 @@ class AnimatedWithChildren extends Animated {
   }
 
   __removeChild(child) {
-    var index = this._children.indexOf(child);
+    const index = this._children.indexOf(child);
     if (index === -1) {
-      console.warn('Trying to remove a child that doesn\'t exist');
+      console.warn( // eslint-disable-line no-console
```

Comment:
I personally dislike loads fo console logs when running tests. I'll add them to an ignore. Hopefully when the API get's updated, some of the deprication warnings can go